deploy/crds/org_v1_che_crd.yaml

```diff
@@ -460,6 +475,32 @@ spec:
                     signed with self-signed cert. So, Che server must be aware of
                     its CA cert to be able to request it. This is disabled by default.
                   type: string
+                singleHostExposureType:
```

Comment:
Since this property only makes sense for the raw kubernetes case (it has only 1 possible value on OpenShift), why not adding it under the K8S section ?
IMO it would make it clearer and would avoid having 2 different default values according to the underlying K8S flavor.

Reply:
I have put it here because of the supposed Ingress v2 coming at some point in the future. But otherwise you're of course right, it would make more sense under k8s.

Reply:
OK. I understand.  Thanks for the explanation.